imports/ui/components/FreshForm.jsx

```diff
@@ -45,6 +49,14 @@ class FreshForm extends Component {
   };
 
   handleSubmit = event => {
+    event.preventDefault();
+    !this.state.add || !this.state.price || !this.state.name
```

Comment:
What happens when we put in somewhere for location but not select something from autofill (then we have address but no lat lon). Is that okay? Will map cards be affected? Do we need check on lat lon? Just curious

Reply:
Also, modal looking great

Reply:
You must make a select by clicking the address :) Otherwise it won't let you submit

Reply:
Actually, I was able to submit without clicking an address from dropdown, just put in some text in address input. So the card does not have lat lon and the submitted card has no address.

Reply:
It doesn't work for me.... 
![Screen Shot 2019-07-06 at 12 15 41 AM](https://user-images.githubusercontent.com/44586417/60753155-5bba9f00-9f83-11e9-92cd-467500fd04c6.png)

Reply:
![image](https://user-images.githubusercontent.com/17561746/60753299-fe275200-9f84-11e9-9580-4f6d383a3910.png)

I was able to reproduce it. I think you just have to press 'Enter' Key and submit shows up enabled.

Reply:
@nicolehli Ohh I see! Yeah upon pressing enter I get the same issue. I'll add the lat & lng check!!!

Reply:
Fixed in the new commit